src/diagnostics/riscv64/disasm-riscv64.cc

```diff
@@ -437,7 +438,6 @@ int Decoder::FormatRvcRegister(Instruction* instr, const char* format) {
         } else if (format[1] == 'f') {
           PrintFPURegister(reg);
         }
-        PrintRegister(reg);
```

Comment:
Why remove this line?

Reply:
Because for this particular formatting option, it will print out c.sub a1, a0, a0a0